static/templates/stream_types.hbs

```diff
@@ -43,7 +43,7 @@
             <select name="stream_message_retention_setting"
               class="stream_message_retention_setting" class="prop-element"
               {{#if disable_message_retention_setting}}disabled{{/if}}>
-                <option value="realm_default">{{t 'Use organization level settings' }} {{realm_message_retention_setting}}</option>
+                <option value="realm_default">{{#tr}}Use organization level settings {org_level_message_retention_setting}{{/tr}}</option>
```

Comment:
Changed this template variable name to `org_level_message_retention_setting` because it gave errors in CI because of word realm present in previous name.

Reply:
I think this is a case where a linter exclude rule would be OK, but this is a fine solution.